android/src/main/java/com/airbnb/android/react/maps/AirMapView.java

```diff
@@ -578,13 +578,16 @@ public boolean dispatchTouchEvent(MotionEvent ev) {
 
         switch (action) {
             case (MotionEvent.ACTION_DOWN):
-                this.getParent().requestDisallowInterceptTouchEvent(true);
+                if (map != null && map.getUiSettings().isScrollGesturesEnabled()) {
+                    this.getParent().requestDisallowInterceptTouchEvent(true);
```

Comment:
@mikelambert All I did was checkout your branch and change `true` to `false, and then the included **Static Map** demo worked as I expected. Did you try testing the demo? On iOS it allows you to drag over the map and it scrolls the ScrollView. In android, before I made this change it would drag the map, *not* the ScrollView. After the change, it would scroll the ScrollView with the map remaining unchanged.